pkg/controller.v1/common/job.go

```diff
@@ -155,6 +142,40 @@ func (jc *JobController) ReconcileJobs(
 		return nil
 	}
 
+	if trainutil.IsJobSuspended(runPolicy) {
+		if err = jc.CleanUpResources(runPolicy, runtimeObject, metaObject, jobStatus, pods); err != nil {
+			return err
+		}
+		for rType := range jobStatus.ReplicaStatuses {
+			jobStatus.ReplicaStatuses[rType].Active = 0
+		}
+		jobStatus.StartTime = nil
```

Comment:
In the MPIJob and Job we don't reset on suspend, but update on resume, for reference:
- https://github.com/kubeflow/mpi-operator/blob/18250f5e6980ce3afbf86359f8aa5fa9ac6cf831/pkg/controller/mpi_job_controller.go#L1027
- https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/job/job_controller.go#L892-L900

Can we do an analogous approach here?

Reply:
> Can we do an analogous approach here?

It's a good suggestion :) 
I agree with doing the approach.

Reply:
Done.